### PR TITLE
feat(config): add -c, --config option

### PR DESCRIPTION
**Pull request by PeterYinusa**
Provide a custom jest config path

> https://github.com/rbardini/jest-it-up/issues/4#issuecomment-756715547 For the custom config path, we could offer the same -c, --config <path> option that [Jest CLI uses](https://jestjs.io/docs/en/cli#--configpath).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -10,12 +10,15 @@ const outputChanges = require('./outputChanges')
 const outputResult = require('./outputResult')
 
 module.exports = async ({
+  config = false,
```
Comment (rbardini) [inline lib/index.js:13]: You can assign a better default here:

```suggestion
  config = 'jest.config.js',
```

File: lib/index.js [hunk 2]
```diff
@@ -10,12 +10,15 @@ const outputChanges = require('./outputChanges')
...
   dryRun = false,
   interactive = false,
   margin = 0,
   silent = false,
 } = {}) => {
-  const configPath = path.resolve(process.cwd(), 'jest.config.js')
+  const configPath = config
+    ? path.resolve(process.cwd(), config)
+    : path.resolve(process.cwd(), 'jest.config.js')
```
Comment (rbardini) [inline lib/index.js:21]: If you apply the suggestions above, you don't need the conditional anymore:

```suggestion
  const configPath = path.resolve(process.cwd(), config)
```

**Review by rbardini — COMMENTED** [review]
Thanks, @PeterYinusa! Sorry for the delayed review, added a couple small suggestions.

This should also resolve #11 💖

File: bin/jest-it-up [hunk 3]
```diff
@@ -9,6 +9,7 @@ program
   .option('-i, --interactive', 'ask for confirmation before applying changes')
   .option('-s, --silent', 'do not output messages')
   .option('-d, --dry-run', 'process but do not change files')
+  .option('-c, --config <path>', 'the path to a Jest config file')
```
[the remark this answers is not in the document]
Comment (PeterYinusa) [reply bin/jest-it-up]: resolved in db585672e17064f8ff5c01d3451fa2ba48e131fd

File: lib/index.js (hunk 2 above)
Comment (PeterYinusa) [reply lib/index.js]: resolved in db585672e17064f8ff5c01d3451fa2ba48e131fd

File: lib/index.js (hunk 1 above)
Comment (PeterYinusa) [reply lib/index.js]: resolved in db585672e17064f8ff5c01d3451fa2ba48e131fd

**Comment by PeterYinusa** [comment]
> Thanks, @PeterYinusa! Sorry for the delayed review, added a couple small suggestions.
> 
> This should also resolve #11 💖

Thanks for taking the time out to review the code, I've made the suggested changes

**Pull request merged**